src/content_sao.cpp

```diff
@@ -725,6 +719,21 @@ u16 LuaEntitySAO::getHP() const
 	return m_hp;
 }
 
+void LuaEntitySAO::hide()
```

Comment:
inline those functions please. And change to hide(bool), we don't need 2 functions

Reply:
Should I change the lua API as well?